django/chat/templates/chat/modals/advanced_qa_modal.html

```diff
@@ -152,7 +152,7 @@ <h5 class="modal-title" id="advanced-qa-modal-title">
               <em>{% trans "(Required) Must include {context_str}, {query_str}, {pre_instructions}, {post_instructions}." %}</em>
             </p>
             <textarea class="form-control form-control-sm" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false" id="qa_prompt_template-modal" data-inputname="qa_prompt_template" onkeypress="updateQaHiddenField(this);" rows="12"></textarea>
-          </div>
+                  </div>
```

Comment:
VS code really hates this thing in particular and moves it an arbitrary distance to the right every time I save. I can't win

Reply:
Curious. Do you have DjLint installed in the devcontainer?

Reply:
Why, yes I do

Reply:
Yep, DjLint was to blame. Weird